TrackingTools/KalmanUpdators/interface/Chi2MeasurementEstimatorBase.h

```diff
@@ -18,12 +18,13 @@ class Chi2MeasurementEstimatorBase : public MeasurementEstimator {
    *  The errors of the trajectory state are multiplied by nSigma 
    *  to define acceptance of Plane and maximalLocalDisplacement.
    */
-  explicit Chi2MeasurementEstimatorBase(double maxChi2, double nSigma = 3.) : 
-    theMaxChi2(maxChi2), theNSigma(nSigma) {}
+  explicit Chi2MeasurementEstimatorBase(double maxChi2, double nSigma = 3., float maxDisp=100.) : 
```

Comment:
for 1-leg case, IIUC, the check is made between extrapolation on one side of the detector to the other.
The effect is the worst for muons with lower number of stations (precision is worse). 
I'm not sure even 3m will always be OK here.
So, probably 10-30m will be about safe.